metab_classes.py

```diff
@@ -281,5 +280,46 @@ def match_authors(self, people: typing.Dict[int, Person]):
                                 (author.name, self.tool_id))
 
 
+class Publication(object):
+    def __init__(self):
+        self.uri = None
+        self.pmid = None
+        self.title = None
+        self.publication_year = None
+        self.datetime = None
+        self.doi = None
+
+    def make_datetime(self):
+        datetime = self.publication_year + "-01-01T00:00:00"
+        self.datetime = datetime
+
+    def get_triples(self):
+        dtv_uri = self.uri + 'dtv'
+        self.make_datetime()
+        rdf = []
+        rdf.append("<{}> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://purl.org/ontology/bibo/Article>".format(self.uri))
+        rdf.append("<{}> <http://www.w3.org/2000/01/rdf-schema#label> \"{}\"^^<http://www.w3.org/2001/XMLSchema#string>".format(self.uri, self.title))
+        rdf.append("<{}> <http://purl.org/ontology/bibo/pmid> \"{}\"^^<http://www.w3.org/2001/XMLSchema#string>".format(self.uri, self.pmid))
+        if self.doi:
+            rdf.append("<{}> <http://purl.org/ontology/bibo/doi> \"{}\"^^<http://www.w3.org/2001/XMLSchema#string>".format(self.uri, self.doi))
+        if self.datetime:
```

Comment:
This will always be true since you run `self.make_datetime()` beforehand.
```suggestion
        if self.publication_year:
```